app/rt1170/led_rollover.conf

```diff
@@ -0,0 +1,5 @@
+# Disable the GPIO heartbeat
+CONFIG_APP_HEARTBEAT=n
+
+# Enable the LED rollover demo
```

Comment:
Remove this file. Because you added `depeonds on !APP_LED_ROLLOVER` to APP_HEARTBEAT, you simple need to enable this appl

`west build ... -- -DAPP_LED_ROLLOVER=y`